### PR TITLE
Small updates and footer responsiveness fix

### PR DESCRIPTION
**Pull request by dojutsu-user**

### CODE REVIEW

**Comment by dojutsu-user** [comment]
Before merging this, I would request to try this in different browser sizes.

File: docs/_ext/hidden_code_block.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# Source: http://scopatz.github.io/hiddencode/
```
Comment (ericholscher) [inline docs/_ext/hidden_code_block.py]: 👍

File: tests/test_ui.py [hunk 2]
```diff
@@ -103,8 +103,7 @@ def test_appending_of_initial_html(selenium, app, status, warning):
                     <span class="bar"></span>
                 </div>
                 <div class="rtd__search__credits">
-                    Search By <a href="https://readthedocs.org/">Read the Docs</a> |
-                    In-Doc Search By <a href="https://readthedocs-sphinx-search.readthedocs.io/en/latest/">readthedocs-sphinx-search</a>
+                    Search by <a href="https://readthedocs.org/">Read the Docs</a> &amp; <a href="https://readthedocs-sphinx-search.readthedocs.io/en/latest/">readthedocs-sphinx-search</a>
```
Comment (ericholscher) [inline tests/test_ui.py]: Why didn't this fail on the PR when we added it?

Comment (dojutsu-user) [reply tests/test_ui.py]: We forgot to update the minified files there.

**Comment by dojutsu-user** [comment]
@ericholscher 
This can be merged now.

**Pull request merged**